wls-wahlvorbereitung-service/src/test/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/rest/unterbrechungsuhrzeit/UnterbrechungsUhrzeitControllerIntegrationTest.java

```diff
@@ -0,0 +1,212 @@
+
+    @Autowired
+    UnterbrechungsUhrzeitRepository unterbrechungsUhrzeitRepository;
+
+    @AfterEach
+    void tearDown() {
+        SecurityUtils.runAs(Authorities.REPOSITORY_DELETE_UNTERBRECHUNGSUHRZEIT);
+        unterbrechungsUhrzeitRepository.deleteAll();
+    }
+
+    @Nested
+    class GetUnterbrechungsUhrzeit {
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT,
+                        Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT }
+        )
+        void dataFound() throws Exception {
+
+            val wahlbezirkIDToFind = "123";
+            val unterbrechungsUhrzeitToFind = new UnterbrechungsUhrzeit();
+            unterbrechungsUhrzeitToFind.setWahlbezirkID(wahlbezirkIDToFind);
+            unterbrechungsUhrzeitToFind.setUnterbrechungsUhrzeit(LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS));
+            unterbrechungsUhrzeitRepository.save(unterbrechungsUhrzeitToFind);
+            val expectedResponseBody = unterbrechungsUhrzeitDTOMapper.toDTO(unterbrechungsUhrzeitModelMapper.toModel(unterbrechungsUhrzeitToFind));
+
+            val request = get("/businessActions/unterbrechungsUhrzeit/" + wahlbezirkIDToFind);
+
+            val response = mockMvc.perform(request).andExpect(status().isOk()).andReturn();
+            val responseBodyAsDTO = objectMapper.readValue(response.getResponse().getContentAsString(), UnterbrechungsUhrzeitDTO.class);
+
+            Assertions.assertThat(responseBodyAsDTO).isEqualTo(expectedResponseBody);
+        }
+
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT,
+                        Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT }
+        )
+        void noDataFound() throws Exception {
+
+            val wahlbezirkIDEmpty = "123";
+
+            val wahlbezirkIDNotEmpty = "456";
+            val unterbrechungsUhrzeitToFind = new UnterbrechungsUhrzeit();
+            unterbrechungsUhrzeitToFind.setWahlbezirkID(wahlbezirkIDNotEmpty);
+            unterbrechungsUhrzeitToFind.setUnterbrechungsUhrzeit(LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS));
+            unterbrechungsUhrzeitRepository.save(unterbrechungsUhrzeitToFind);
+
+            val request = get("/businessActions/unterbrechungsUhrzeit/" + wahlbezirkIDEmpty);
+
+            val response = mockMvc.perform(request).andExpect(status().isNoContent()).andReturn();
+
+            Assertions.assertThat(response.getResponse().getContentAsString()).isEmpty();
+        }
+    }
+
+    @Nested
+    class PostUnterbrechungsUhrzeit {
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT,
+                        Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT }
+        )
```

Comment:
Das Read-Recht wird erst nach dem mockMVC.perform benötigt und dort wirt es auch explizit gesetzt
```suggestion
        @WithMockUser(
                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT }
        )
```